x/nodes/keeper/valStateChanges.go

```diff
@@ -261,6 +261,15 @@ func (k Keeper) ValidateEditStake(ctx sdk.Ctx, currentValidator, newValidtor typ
 				return types.ErrUnequalOutputAddr(k.Codespace())
 			}
 		}
+
+		// Delegators can be set/edited only if 1) the feature has been activated
+		// AND 2) the message is signed by the operator address.
+		if k.Cdc.IsAfterDelegatorUpgrade(ctx.BlockHeight()) &&
+			!types.CompareStringMaps(currentValidator.Delegators, newValidtor.Delegators) &&
+			!signer.Equals(currentValidator.Address) {
+			return types.ErrDisallowedOutputAddressEdit(k.Codespace())
```

Comment:
Should this be a `DelegatorEdit` error?

Reply:
I was lazy not to introduce a new error.  Added it.